CSS/css_completions.py

```diff
@@ -84,19 +79,19 @@ def get_properties():
             'bisque', 'black', 'blanchedalmond', 'blue', 'blueviolet', 'brown',
             'burlywood', 'cadetblue', 'chartreuse', 'chocolate', 'coral',
             'cornflowerblue', 'cornsilk', 'crimson', 'cyan', 'darkblue',
-            'darkcyan', 'darkgoldenrod', 'darkgray', 'darkgrey', 'darkgreen',
+            'darkcyan', 'darkgoldenrod', 'darkgray', 'darkgreen',
```

Comment:
do I understand correctly that the English spelling of `grey` is deprecated? :)

Reply:
I don't recall where I got that from. Too long ago.